### PR TITLE
Fix graphite migration

### PR DESCRIPTION
**Pull request by zmc**

### CODE REVIEW

File: ansible/roles/ceph-grafana/tasks/configure_graphite_web.yml [hunk 1]
```diff
@@ -1,7 +1,14 @@
 ---
 - name: Create Graphite DB tables
-  command: /usr/bin/graphite-manage syncdb --noinput
+  command: /usr/bin/graphite-manage migrate --noinput
   become_user: "{{ graphite.unix_user[ansible_pkg_mgr] }}"
+  register: migrate_cmd
+  failed_when: migrate_cmd.rc != 0 and 'table "django_content_type" already exists' not in migrate_cmd.stderr
+
+- name: Run Graphite migrations in necessary
```
Comment (b-ranto) [inline ansible/roles/ceph-grafana/tasks/configure_graphite_web.yml]: Was this supposed to be 'if necessary'?

File: ansible/roles/ceph-grafana/tasks/configure_graphite_web.yml [hunk 2]
```diff
@@ -1,7 +1,14 @@
 ---
 - name: Create Graphite DB tables
-  command: /usr/bin/graphite-manage syncdb --noinput
+  command: /usr/bin/graphite-manage migrate --noinput
```
Comment (b-ranto) [inline ansible/roles/ceph-grafana/tasks/configure_graphite_web.yml]: Did you test this on clean install? (well, purge + reinstall)

Comment (zmc) [reply ansible/roles/ceph-grafana/tasks/configure_graphite_web.yml]: I'll do that shortly

Comment (zmc) [reply ansible/roles/ceph-grafana/tasks/configure_graphite_web.yml]: ok - I'm satisfied.

**Pull request merged**